skimage/novice/_novice.py

```diff
@@ -326,6 +326,9 @@ def save(self, path):
         path : str
             Path (with file extension) where the picture is saved.
         """
+        if (self.array.ndim >= 3 and self.array.shape[-1] >= 4 and
```

Comment:
This check should be exact (`shape[-1] == 4`, since you rely on `[:-1]` to produce a `(3,)` in the last axis.  I think the first can be exact as well, `array.ndim == 3`.